lib/buttons-and-links/__tests__/KButton.spec.js

```diff
@@ -67,9 +67,102 @@ describe('KButton', () => {
   });
 
   describe.visual('KButton Visual Tests', () => {
-    it('renders correctly with default props', async () => {
-      await renderComponent('KButton', { text: 'Test Button' });
-      await takeSnapshot('KButton - Default');
+    describe('renders correctly with different appearances', () => {
+      it('renders correctly as primary raised button', async () => {
+        await renderComponent('KButton', {
+          text: 'Raised Button',
+          primary: true,
+          appearance: 'raised-button',
+        });
+        await takeSnapshot('KButton - Primary Raised Button', { widths: [400], minHeight: 512 });
```

Comment:
As we will probably have the same sizes for (almost) all the screenshots of a single component, I think we can make this a constant object defined in the context of the describe.visual block and use the same options object thoughout the tests.

Reply:
I've made the changes as requested ✅

Reply:
Thank you!!